peer.go

```diff
@@ -362,6 +363,17 @@ func (p *peer) Start() error {
 	go p.channelManager()
 	go p.pingHandler()
 
+	// Now that the peer has started up, we send any channel sync messages
+	// that must be resent for borked channels.
+	if len(msgs) > 0 {
+		peerLog.Infof("Sending %d channel sync messages to peer after "+
+			"loading active channels", len(msgs))
+		if err := p.SendMessage(true, msgs...); err != nil {
```

Comment:
We should remove `resendChanSyncMsg` to prevent sending it twice upon reconnections.

Reply:
`resendChanSyncMsg` is only used for closed channels, here we send for borked, open channels.

Reply:
Any reason to not unify those with these?